application/views/parts/footer.php

```diff
@@ -1,5 +1,5 @@
 <center>
-	<img src="<?php echo public_url('site/images/bank.png'); ?>" /> 
+	<img src="<?php echo public_url("site/images/bank.png"); ?>" /> 
```

Comment:
__[code_sniffer]__
* Single-quoted strings should be used unless it contains variables, special chars like \n or single quotes.

<!-- Comment by [SideCI](https://sideci.com) -->